ocs_ci/ocs/cluster.py

```diff
@@ -1362,6 +1362,66 @@ def check_ceph_osd_tree_after_node_replacement():
     return True
 
 
+def silence_ceph_osd_crash_warning(osd_pod_name):
+    """
+    Silence the osd crash warning of a specific osd pod
+
+    Args:
+        osd_pod_name (str): The name of the osd pod which we need to
+            silence the crash warning
+
+    Returns:
+        bool: True if it found the osd crash with name 'osd_pod_name'. False otherwise
+
+    """
+    ct_pod = pod.get_ceph_tools_pod()
+    new_crash_objects_list = ct_pod.exec_ceph_cmd(ceph_cmd="ceph crash ls-new")
+    for crash_obj in new_crash_objects_list:
+        if crash_obj.get("utsname_hostname") == osd_pod_name:
+            logger.info(f"Found osd crash with name {osd_pod_name}")
+            obj_crash_id = crash_obj.get("crash_id")
+            crash_info = ct_pod.exec_ceph_cmd(
+                ceph_cmd=f"ceph crash info {obj_crash_id}"
+            )
+            logger.info(f"ceph crash info: {crash_info}")
+
+            logger.info("silence the osd crash warning")
```

Comment:
Can unite the two `logger.info` (lines 1386,1388)

Reply:
I think it will be better if we separate the logs because it's too different subjects.